web/static/js/graph_template.handlebar

```diff
@@ -1,5 +1,5 @@
         <div id="graph_wrapper{{id}}" class="graph_wrapper">
-          <form action="/api/query_range" method="GET" class="query_form form-inline">
+          <form action="{{ getUrlPrefix }}api/query_range" method="GET" class="query_form form-inline">
```

Comment:
Shouldn't this be `URL_PREFIX` too, since it's not a Go template?

Did you try graphing stuff, and does it still work? Actually I'd have to check how handlebar templates work and whether these `{{ ... }}` expressions may include any JS variable, or only the ones that were explicitly passed to the template.